geppetto.js/geppetto-ui/src/3d-canvas/threeDEngine/MeshFactory.js

```diff
@@ -280,6 +284,16 @@ export default class MeshFactory {
       threeObject = this.loadThreeOBJModelFromNode(node);
       this.complexity++;
       break;
+    case GEPPETTO.Resources.GLTF:
+      threeObject = await this.loadThreeGLTFModelFromNode(node);
+      this.complexity++;
+      break;
+    case GEPPETTO.Resources.DRC:
+      threeObject = await this.loadThreeDRCModelFromNode(node);
+      this.complexity++;
+      break;
+    default:
+      console.error("Invalid node.eClass")
```

Comment:
Would also log here the node or at least the eClass to easily see what the invalid class/node was